roxie/ccd/ccdfile.cpp

```diff
@@ -1059,9 +1065,12 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
             const char *kind = fdesc.queryKind();
             if (kind && streq(kind, "key"))
                 isKey = true;
+            StringBuffer planeName;
+            fdesc.getClusterLabel(pdesc->copyClusterNum(0), planeName);
+            blockedSize = getBlockedFileIOSize(planeName);
```

Comment:
This needs to use a different variable/property of the storage plane.  It is likely that we want to use 1MB when reading an index sequentially, but only 32/64K when reading it randomly.

Reply:
hm, ok, perhaps pass a accessmode type flag to getBlockedFileIOSize, with it determining (via new prop. if available)